src/style.css

```diff
@@ -1 +1,24 @@
 /* styles go here */
+
+legend {
+  margin: 0;
+}
+
+label {
+  display: block;
+  margin: 10px 0;
+  margin-bottom: 0;
+}
+
+fieldset {
+  margin: 20px 2px;
+  margin-top: 0;
+}
+
+button {
+  margin: 0;
+}
+
```

Comment:
Use classname

Reply:
you already created class for label <label class="top">

in css

`.top {styles}`

Reply:
still need to be fixed
don't style tags, use classes